util/regexp-assemble/regexp-942120.data

```diff
@@ -1,3 +1,21 @@
+##! This is a data file used to generate a regular expression for a CRS rule. 
```

Comment:
Looks good! The ignore case flag is written in a new way. This should be replaced in the rule file.
`(?i)` instead of `(?i:...)`

Reply:
Done.